docs/docs/migrations.md

```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
     allowNull: false
   }
 )
+
+// or with an explicit schema:
+
+queryInterface.addColumn({
+    tableName: 'nameOfAnExistingTable',
```

Comment:
You forgot to change these. Use a real world example here for tableName, schema and attribute